var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -77,14 +77,10 @@ class Openspeedshop(CMakePackage):
                          to point to target build.")
     variant('cuda', default=False,
             description="build with cuda packages included.")
-    variant('useqt4gui', default=False,
-            description="build with Qt4/Qt5 based gui package enabled. \
-                         Do not build older Qt3 gui")
-    variant('rtfe', default=False,
-            description="build for clusters heterogeneous processors \
-                         on fe/be nodes.")
-    variant('build_type', default='None', values=('None'),
-            description='CMake build type')
```

Comment:
You need to keep this variant. This is what controls the CMake build type now, we got rid of the `build_type` function.